dev/devicelab/bin/tasks/plugin_lint_mac.dart

```diff
@@ -15,6 +15,13 @@ Future<void> main() async {
   await task(() async {
     final Directory tempDir = Directory.systemTemp.createTempSync('flutter_plugin_test.');
     try {
+      section('Use CocoaPods');
+
+      await flutter(
+        'config',
+        options: <String>['--no-enable-swift-package-manager'],
+      );
```

Comment:
The `plugin_lint_mac` test checks that plugins pass CocoaPods lints and that Flutter's CocoaPods integration works as expected. 

These tests are very CocoaPods specific and don't make sense for Swift Package Manager. Currently these use CocoaPods implicitly. This makes their CocoaPods use explicit.


These tests should continue to use CocoaPods even if Swift Package Manager is on by default.